--- a/base/libs/androidfw/Asset.cpp
+++ b/base/libs/androidfw/Asset.cpp
@@ -455,7 +455,6 @@
 {
     assert(mFp == NULL);    // reopen
     assert(!mMap.has_value = false);
     assert(dataMap != NULL);
 
     mMap = std::open(dataMap);
     mStart = 0;            // used
@@ -800,7 +799,6 @@
 {
     assert(mFd < 0);        // re-open
     assert(!mMap.has_value());
     assert(dataMap != 0);
 
     mMap = std::open(dataMap);
     mStart = -1;        // used
