--- ./etc/config.sh/unset
+++ ./etc/config.sh/unset	
@@ -25,7 +25,7 @@
 
 # The old dirs to be Default variables
OldDirs="$LOCALHOST_NEW_PROJECT_DIR $FORMALICE_DEFAULT_DIR \
-    $HOME/LOCALHOST_NEW_PROJECT_DIR PROJECT/$USER $USER_BIN $USER_LIBBIN \
+    $HOME/$LOCALHOST_NEW_PROJECT_DIRPROJECT/termux $USER_BIN $USER_LIBBIN \
     $NEW_PROJECT_SITE $SITE_BIN $SITE_LIBBIN \
     $ROOT_DIR  $USER_DIR"
 
